examples/opcuaDeviceShifu/opcua_deploy/opcua_deviceshifu_deployment.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         - name: edgedevice-certificate
           mountPath: "/etc/edgedevice/certificate"
           readOnly: true
+        - name: deviceshifu-secret
+          mountPath: /etc/edgedevice/secret
+          readOnly: true
```

Comment:
instead of creating the secret, we can use https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables to load them into ENV, this will allow us to use existing codes

Reply:
Cool. For Haikvision driver, it is true and I have rolled back the code. However, for OPCUA or other deviceshifu, the original code does not seem to directly read from the environment variable, so loading from ENV or secret mount file would require similar modifications.